ucfcs/GrowthPlus
# UI refactor

The pr updates the UI to be responsive in particular to smaller sized phones. I made a new layout folder called layout-sw340dp which means that any phone with the minimum with of 340dp or larger will use those xmls but anything smaller as well as the default layouts are in the layout folder. 

TEST:
- run this branch on a device that has a density equal to hdpi or less. (Nexus One or Nexus S are the main ones)
- also run this branch on the emulator device size that you typically use
- make sure the app isn't breaking and that the layouts look good (there aren't many changes for devices larger than the Nexus S/One, I did make the child cards in the parent portal and main activity larger)
- there should not be any overlapping stuff in the UI and everything should look pretty clean 

Closes #117
